src/components/profile/ProfileService.js

```diff
@@ -304,6 +383,8 @@ goog.require('ga_urlutils_service');
           }
           var x = d3.scale.linear().range([0, width]);
           var y = d3.scale.linear().range([height, 0]);
+          this.diff = this.elevDiff(data);
```

Comment:
On window's resize dats is null, that'y why the profile is not resizing. Move the operation on the previous `else` block